stock_prices/stock_prices.py

```diff
@@ -2,8 +2,14 @@
 
 import argparse
 
-def find_max_profit(prices):
-  pass
+def find_max_profit (stock_prices):
+  profit = []
+  for i in range(0, len(stock_prices) - 1):
```

Comment:
you're running two for loops here, making your solution O(n**2). it works fine! but see if you can bring your solution down to O(n) after you're done with MVP